Assets/PatchKit Patcher/Scripts/UI/Languages/PatcherLanguages.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Newtonsoft.Json;
+using PatchKit.Unity.Patcher.Debug;
+using PatchKit.Unity.Utilities;
+using UnityEngine;
+
+namespace PatchKit.Unity.UI.Languages
+{
+    public class PatcherLanguages : MonoBehaviour
+    {
+        public static PatcherLanguages Instance;
+
+        public static Dictionary<String, String> Fields;
+
+        [SerializeField] 
+        string defaultLanguage = "en";
+
+        private static readonly DebugLogger DebugLogger = new DebugLogger(typeof(PatcherLanguages));
+
+        void Awake ()
+        {
+            if (Instance == null) {
+                Instance = this;
+                DontDestroyOnLoad (gameObject);
+            } else {
+                Destroy (gameObject);
+            }
+
+            LoadLanguage ();
+        }
+
+
+        void LoadLanguage ()
```

Comment:
Can be converted to "lazy loading".

Meaning:
- if `Fields` is not null then the method runs as it is done right now
- otherwise, it just returns as language is already loaded

Reply:
done